docs/HyperIndex/terms-of-service.md

```diff
@@ -29,15 +29,27 @@ _The fine print: Please note these terms are intended to protect us, where adver
 
 4. **Payments and Refunds**
 
-   Our products are currently provided on a free-to-use basis. We reserve the right to introduce charges for these services at our discretion. Any pricing changes will be communicated to users in advance.
+   Our services are provided on a paid basis. By subscribing or making a payment, you agree to the applicable fees and billing terms.
+
+   Billing & Payment: Fees are charged upfront based on the selected plan. Any additional unit fees exceeding the included base usage will be billed separately at the end of the calendar month. Payments must be made via the payment methods we support.
+
+   No Refunds: All payments are non-refundable. We do not provide refunds or credits for any unused service, partial subscription periods, downgrades, or cancellations.
+
+   Pricing Changes: We reserve the right to modify our pricing at any time. Any changes will be communicated in advance and will take effect at the start of the next billing cycle.
```

Comment:
Should we add an exact time, for example, "two weeks before the next billing cycle"?

Reply:
I'd prefer not to create arbitrary commitments, I think in this case we'd likely update with more than two weeks but the above wording affords more flexibility.